src/sgame/sg_weapon.cpp

```diff
@@ -40,7 +40,7 @@ static bool TakesDamages( gentity_t const* ent )
 {
 	return false
 		|| Entities::IsAlive( ent )
-		|| ( ent && ent->s.eType == entityType_t::ET_MOVER )
+		|| ( ent && ent->s.eType == entityType_t::ET_MOVER && ent->health > 0 )
```

Comment:
while I think this particular style is an antipattern in the code, I won't require that you change it.

Reply:
wdym?

Reply:
the 
```
false 
|| cond1
|| cond2
;
```

Reply:
Well, I'm ok to change it, but I would appreciate if you could elaborate on why it's an anti-pattern. According from wikipedia, an anti-pattern is something that: "has more bad consequences than good ones". Tell me, what are the bad consequences of this?

Reply:
I guess what I meant by anti pattern is that it's not really used anywhere in the code and I don't think it really provides anything over something like:
```
cond1 || cond2
```
or 
```
cond1
|| cond2
|| cond3;
```

but as I said, I don't want to make this a big enough deal to change, especially when we don't have a codified style to enforce :(